service-common/src/actix_auth/authentication/mod.rs

```diff
@@ -51,7 +53,7 @@ impl AuthN {
     async fn authenticate(
         &self,
         credentials: Credentials,
-    ) -> Result<UserInformation, ServiceError> {
+    ) -> Result<(UserInformation, Option<i64>), ServiceError> {
```

Comment:
I would say that if we do the work of converting it then let's do it all the way to an `Instant` because a duration may be misused as well. However it adds the conversion cost for every requests when only the WS service makes use of it